woocommerce/woocommerce-gutenberg-products-block
# Enhancement: Changing tax_query field to term_id

This PR changes the field from the tax_query defined in ProductTag class, so it's compatible with the [core](https://developer.wordpress.org/reference/classes/wp_query/#taxonomy-parameters) definition of WP_Query taxonomy parameters. 

The field possible values are `term_id`, `name`, `slug` or `term_taxonomy_id`. The default value is `term_id`.

Fixes #6584 

### Testing
1. Create a page and add a `Products by Tag` block.
2. Check that is rendered correctly in the editor.
3. Save the page and check that is rendered correctly in the frontend.

### Performance Impact

Should be none, since the query already defaults to `term_id` by default

### Changelog

> Enhancement: Replace the ProductTag tax_query field to be the term_id instead of the id.
